x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/notification/slack/message/SlackMessageTests.java

```diff
@@ -49,7 +49,7 @@ public void testToXContent() throws Exception {
         }
         String icon = randomBoolean() ? null : randomAlphaOfLength(10);
         String text = randomBoolean() ? null : randomAlphaOfLength(50);
-        Attachment[] attachments = randomBoolean() ? null : new Attachment[randomIntBetween(0, 2)];
+        Attachment[] attachments = !(randomBoolean() && text == null) ? null : new Attachment[randomIntBetween(0, 2)];
```

Comment:
we have a convention to not use the `!` operator in statements like this. Please use `== false` instead.

`(text != null && randomBoolean()) ? null : new .....`

This should also work

Reply:
Out of curiosity, why?

I'll make this change.